concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -1,909 +1,909 @@
+Marrison-1967-909-515	515	no	1269	NO	2
+Marrison-1967-909-516	516	noise	1182	NOISE	2
+Marrison-1967-909-517	517	noon	12	MIDDAY	2
+Marrison-1967-909-518	518	north	1598	NORTH	2
+Marrison-1967-909-519	519	nose	1221	NOSE	2
+Marrison-1967-909-520	520	not	1240	NOT	2
+Marrison-1967-909-521	521	now	1376	NOW	2
+Marrison-1967-909-522	522	oath	1712	OATH	2
+Marrison-1967-909-523	523	obey	1635	OBEY	2
+Marrison-1967-909-524	524	oil	1484	OIL (ORGANIC SUBSTANCE)	2
+Marrison-1967-909-525	525	old (of persons)	2112	OLD (AGED)	4
+Marrison-1967-909-526	526	old (of things)	2113	OLD (USED)	4
+Marrison-1967-909-527	527	old man	406	OLD MAN	2
+Marrison-1967-909-528	528	old woman	907	OLD WOMAN	2
+Marrison-1967-909-529	529	on	2379	UP OR ABOVE	4
+Marrison-1967-909-530	530	onion	2366	ONION	2
+Marrison-1967-909-531	531	only	1580	ONLY	2
+Marrison-1967-909-532	532	open	1156	OPEN	2
+Marrison-1967-909-533	533	orange	2627	ORANGE (FRUIT)	2
+Marrison-1967-909-534	534	order	1128	ORDER	4
+Marrison-1967-909-535	535	orphan	1149	ORPHAN	2
+Marrison-1967-909-536	536	other	197	OTHER	2
+Marrison-1967-909-537	537	otter	15	OTTER	2
+Marrison-1967-909-538	538	outside	762	OUTSIDE	2
+Marrison-1967-909-539	539	owl	735	OWL	2
+Marrison-1967-909-540	540	owner	1036	OWNER	2
+Marrison-1967-909-541	541	pain	1783	PAIN	2
+Marrison-1967-909-542	542	palm (hand)			4
+Marrison-1967-909-543	543	palm (of hand)	1183	PALM OF HAND	2
+Marrison-1967-909-544	544	panji (bamboo stake)			
+Marrison-1967-909-545	545	pass by	558	PASS BY	2
+Marrison-1967-909-546	546	pay	718	PAY	2
+Marrison-1967-909-547	547	peace	1338	PEACE	2
+Marrison-1967-909-548	548	peacock			
+Marrison-1967-909-549	549	pearl			
+Marrison-1967-909-550	550	pen	794	PEN	2
+Marrison-1967-909-551	551	people	789	PEOPLE	2
+Marrison-1967-909-552	552	person	683	PERSON	2
+Marrison-1967-909-553	553	pierce	398	PIERCE	2
+Marrison-1967-909-554	554	pig (domestic)	1337	PIG	4
+Marrison-1967-909-555	555	pig (wild)	3154	WILD BOAR	4
+Marrison-1967-909-556	556	pig(wild)			4
+Marrison-1967-909-557	557	pigeon	1853	DOVE	2
+Marrison-1967-909-558	558	pillow	1044	PILLOW	2
+Marrison-1967-909-559	559	pipe (tobacco)	1124	PIPE	4
+Marrison-1967-909-560	560	place	723	PLACE (POSITION)	2
+Marrison-1967-909-561	561	plains	652	PLAIN	4
+Marrison-1967-909-562	562	plate	480	PLATE	2
+Marrison-1967-909-563	563	play (a game)	3125	PLAY (GAME)	4
+Marrison-1967-909-564	564	play (music)	3124	PLAY (INSTRUMENT)	4
+Marrison-1967-909-565	565	poison	655	POISON	2
+Marrison-1967-909-566	566	pool			
+Marrison-1967-909-567	567	poor	1674	POOR	2
+Marrison-1967-909-568	568	porcupine	2510	PORCUPINE	2
+Marrison-1967-909-569	569	post (of house)	281	POST	4
+Marrison-1967-909-570	570	pot	1462	POT	2
+Marrison-1967-909-571	571	power	2526	POWER	2
+Marrison-1967-909-572	572	praise	1687	PRAISE	2
+Marrison-1967-909-573	573	pray	24	PRAY	2
+Marrison-1967-909-574	574	pregnant	1123	PREGNANT	2
+Marrison-1967-909-575	575	prepare	2909	PREPARE	2
+Marrison-1967-909-576	576	press	1913	PRESS	2
+Marrison-1967-9
```

Comment:
can you find out the part of speech? It may be more info in the diss. If not, leave as is: rain or rain is unspecific...